test/extended/cmd.sh

```diff
@@ -57,7 +57,9 @@ install_registry
 wait_for_registry
 docker_registry="$( oc get service/docker-registry -n default -o jsonpath='{.spec.clusterIP}:{.spec.ports[0].port}' )"
 
+os::test::junit::declare_suite_start "extended/cmd"
 
+os::test::junit::declare_suite_start "extended/cmd/new-app"
```

Comment:
Sure, just wanted to make sure the movement wasn't erroneous or that a test was missing.